nuxt.config.ts

```diff
@@ -24,5 +29,9 @@ export default defineNuxtConfig({
   },
   colorMode: {
     preference: 'light'
+  },
+  tailwindcss: {
+    injectPosition: { after: LikeCoinWalletConnectorCSSPath },
+    cssPath: '~/assets/css/tailwind.css',
```

Comment:
<img width="783" alt="image" src="https://github.com/likecoin/nft-book-press/assets/17264716/6a39bec0-76ed-444b-a848-0d4fcb31b5d5">

`injectPosition` should be under `cssPath`?